README.md

```diff
@@ -180,6 +180,8 @@ sccache defaults to using local disk storage. You can set the `SCCACHE_DIR` envi
 
 The default cache size is 10 gigabytes. To change this, set `SCCACHE_CACHE_SIZE`, for example `SCCACHE_CACHE_SIZE="1G"`.
 
+The local storage only supports a single concurrent build. Multiple concurrent builds will race and cause spurious build failures.
```

Comment:
That's not technically true. You can do as many concurrent builds as you want. As a matter of fact, practically speaking, that's what happens when doing "a build".

What's not supported is having multiple sccache servers (e.g. sccache --start-server) pointing to the same cache directory.

Reply:
Done